docs/docs/en/guide/installation/standalone.md

```diff
@@ -19,9 +19,16 @@ If you are a new hand and want to experience DolphinScheduler functions, we reco
 There is a standalone startup script in the binary compressed package, which can be quickly started after extraction. Switch to a user with sudo permission and run the script:
 
 ```shell
-# Extract and start Standalone Server
+# Extract Standalone Server
 tar -xvzf apache-dolphinscheduler-*-bin.tar.gz
 cd apache-dolphinscheduler-*-bin
+```
+
+### Modify dolphinscheduler_env.sh
```

Comment:
It's better not to modify config file. Let the user do `export DATABASE=h2` before running the daemon script.

Reply:
```export DATABASE=h2``` before execute daemon script will be overwrite by ```export DATABASE``` in the ```dolphinscheduler-daemon.sh```. I think we can use the method proposed by @zhongjiajie  in wechat. What do you think?

Reply:
> ```export DATABASE=h2``` before execute daemon script will be overwrite by ```export DATABASE``` in the ```dolphinscheduler-daemon.sh```.

Why do you think so? The mechanism by design is to allow not modifying config file to override config in command line. If this doesn't work it is a bug

Reply:
> It's better not to modify config file. Let the user do `export DATABASE=h2` before running the daemon script.

that a good idea too, before that I think we should add option to `dolphinscheduler-deamon.sh` like `--no-overwrite` or some other things, to determin whether we should overwrite env file.

Reply:
`export DATABASE=h2` before daemon script must work otherwise it's a bug of our script. I can take a look at that later

Reply:
For example
`shell
export DATABASE=${DATABASE:-postgresql}
sh ./bin/dolphinscheduler-daemon.sh start standalone-server
`

The `script/dolphinscheduler-daemon.sh's` `overwrite_server_env()` will overwrite the `standalone-server/conf/dolphinscheduler_env.sh```. And `standalone/bin/start.sh` will `source "$DOLPHINSCHEDULER_HOME/conf/dolphinscheduler_env.sh"`. It will overwrite the `export DATABASE=${DATABASE:-postgresql}` from the beginning.